### ISSUE TITLE
Direct link to download OVFTool?

### ISSUE THREAD

**Issue by mmumshad** [djui/docker-ovftool#1]
Hi,

I would like to embed downloading the OVFTool directly from the website inside my Dockerfile. Would you know if there one such link?

Thanks

**Comment by djui** [comment]
No, I don't know of any direct download link. The link I provide in the README required a "my vmware" account.
